docs/dev-guide/changes.md

```diff
@@ -0,0 +1,231 @@
+the user can specify the number of choosers in each chunk explicitly, either as an integer
+number of choosers per chunk, or as a fraction of the overall number of choosers.
+This is done by setting the `explicit_chunk` configuration setting in the model
+component's settings.  If the `explicit_chunk` value is 0 or missing, then no
+chunking is applied for that component.  The `explicit_chunk` values are ignored
+if the `chunk_training_mode` is not set to `explicit`.
+Refer to each model component's configuration documentation for details.
+
+Refer to code updates that implement explicit chunking for accessibility in
+[PR #759](https://github.com/ActivitySim/activitysim/pull/759), for
+vehicle type choice, non-mandatory tour frequency, school escorting, and
+joint tour frequency in  [PR #804](https://github.com/ActivitySim/activitysim/pull/804),
+and all remaining interaction-simulate components in
+[PR #870](https://github.com/ActivitySim/activitysim/pull/870).
+
+### Automatic dropping of unused columns
+
+Variables that are not used in a model component are now automatically dropped
+from the chooser table before the component is run. Whether a variable is deemed
+as "used" is determined by a text search of the model component code and specification
+files for the variable name.  Dropping unused columns can be disabled by setting
+[`drop_unused_columns`](activitysim.core.configuration.base.ComputeSettings.drop_unused_columns)
+to `False` in the [`compute_settings`](activitysim.core.configuration.base.ComputeSettings)
+for any model component, but by default this setting is `True`, as it can result in a
+significant reduction in memory usage for large models.
+
+Dropping columns may also cause problems if the model is not correctly configured.
+If it is desired to use this feature, but some required columns are being dropped
+incorrectly, the user can specify columns that should not be dropped by setting the
+[`protect_columns`](activitysim.core.configuration.base.ComputeSettings.protect_columns)
+setting under [`compute_settings`](activitysim.core.configuration.base.ComputeSettings).
+This allows the user to specify columns that should not be dropped, even if they are
+not apparently used in the model component.
+
+Code updates to drop unused columns are in
+[PR #833](https://github.com/ActivitySim/activitysim/pull/833) and to protect
+columns in [PR #871](https://github.com/ActivitySim/activitysim/pull/871).
+
+### Automatic conversion of string data to categorical
+
+Version 1.3 introduces a new feature that automatically converts string data
+to categorical data.  This reduces memory usage and speeds up processing for
+large models.  The conversion is done automatically for string columns
+in most chooser tables.
+
+See code updates in [PR #782](https://github.com/ActivitySim/activitysim/pull/782)
+and [PR #863](https://github.com/ActivitySim/activitysim/pull/863)
+
```

Comment:
Suggest adding here that - There is also an optional downcasting of numeric data available if the user want to further reduce memory. But we suggest using it with caution and it is defaulted to `False` (not downcasting numerics) because: 1) changing the precision of numeric data could cause results to change slightly and impact a previous calibrated model result, and 2) downcasting to lower byte data types, e.g., int8, can cause numeric overflow in downstream components if the numeric variable is used in math calculations such as squaring, that would result in values beyond the int8 limit. We recommend users to review their model specification and review the result change if they choose to downcast numeric data. To downcast, users can set the `downcast_int` and `downcast_float` to `True` in the top level model configuration file (typically `settings.yaml`)